src/OrleansSQLUtils/Storage/DbExtensions.cs

```diff
@@ -70,19 +71,20 @@ public static class DbExtensions
         /// <param name="parameterName">The name of the parameter.</param>
         /// <param name="value">The value of the parameter.</param>
         /// <param name="size">The size of the parameter value.</param>
+        /// <param name="dbType">the <see cref="DbType"/> of the parameter.</param>
         /// <returns>A parameter created using the given arguments.</returns>
-        public static IDbDataParameter CreateParameter<T>(this IDbCommand command, ParameterDirection direction, string parameterName, T value, int? size = null)
+        public static void AddParameter<T>(this IDbCommand command, string parameterName, T value, ParameterDirection direction = ParameterDirection.Input, int? size = null, DbType? dbType=null)
```

Comment:
I wondered about this, I'll keep the `CreateParameter` method and add an `AddParameter` one.